src/main/riddl/Members/members.riddl

```diff
@@ -313,21 +292,13 @@ context MembersContext  is {
                     and "set entity lastModifiedOn to today()"
                     and "yield event MemberInfoUpdated"
                 }
-                on command ActivateMember {
-                  then error "Cannot activate a member that is already activated"
-                }
-                on command InactivateMember {
-                  then "morph Member entity to Inactive state"
-                }
-                on command SuspendMember {
-                  given "entity has suspensionReason non-empty"
-                  then "morph Member entity to Suspended state"
-                  and "set entity lastModifiedBy to actingMember from message"
-                  and "set entity lastModifiedOn to today()"
-                  and "yield event MemberSuspended"
+                on command UpdateMemberStatus {
+                    then "morph entity Member to state provided in UpdateMemberStatus"
+                    and "set Meta data to reflect the change"
+                    and "yield event MemberStatusUpdated"
```

Comment:
Again, I would prefer the individual handling of these three commands you have deleted.  If you want to keep it DRY, create a function in this entity to handle the common processing part.